xdsl/dialects/llvm.py

```diff
@@ -474,6 +475,64 @@ def print(self, printer: Printer) -> None:
         printer.print(self.lhs.type)
 
 
+class ArithmeticBinOpExact(IRDLOperation, ABC):
+    """Class for arithmetic binary operations that use an exact flag."""
+
+    T: ClassVar = VarConstraint("T", BaseAttr(IntegerType))
+
+    lhs = operand_def(T)
+    rhs = operand_def(T)
+    res = result_def(T)
+    isExact = opt_prop_def(BoolAttr)
```

Comment:
This is the bridge we use to make sure that our Python code conforms to snake_case and that the interop with MLIR works, I'd recommend using it in this case:
```suggestion
    is_exact = opt_prop_def(BoolAttr, prop_name="isExact")
```